src/components/ApproachPage/HowWeEngage.tsx

```diff
@@ -1,10 +1,10 @@
 import { css } from '@emotion/react';
 import type { FC } from 'react';
 
-import { Heading, IconCard, Link } from '~/components';
+import { Heading, Link } from '~/components';
```

Comment:
👇 We should grab `CardGrid` and `IconCard` here.

```suggestion
import { CardGrid, Heading, IconCard, Link } from '~/components';
```